text/0266-move-oteps-to-spec.md

```diff
@@ -0,0 +1,66 @@
+# Move OTEPS to the Specification repository.
+
+Let's move OTEP documentation and PRs back into the github.com/open-telemetry/opentelemetry-specification repository.
+
+## Motivation
+
+Moving OTEPs back into the specification solves two main issues:
+
+- Maintaining its tooling infrastructure (currently woefully out of date)
+- Bringing it into the existing triage and voting process currently within the
+  Specification.
+
+## Explanation
+
+Originally, OTEPs were kept as a separate repository to keep disjoint/disruptive designs as a separate repository.  There are a few differences between a normal PR and an OTEP:
+
+- OTEPs are expected to be directional and subject to change when actually entered into the specification.
+- OTEPs require more approvals than specification PRs
+- OTEPs have different PR worklfows (whether due to accidental omission or conscious decision), e.g. staleness checks, linting.
+
+As OpenTelemetry is stabilizing, the need for OTEPs to live outside the specification is growing less, and we face challenges like:
+
+- Keeping OTEP tooling up to date
+- Advertising the repositories existence
+  - New contributors to OpenTelemetry often can't find recorded decision that exist in OTEPs.
+  - Getting reviews from folks used to checking the Specification repository, but not the less-frequently-worked-on OTEP repository.
+
+To solve these, let's move OTEPs into a directory within the [specification repository](github.com/open-telemetry/opentelemetry-specification).
+We would also update all tooling and expected reviews to match existing standards for OTEPs.  Given the maintainers of OTEPs are the same as
+maintainers of the specification, this should not change the bar for acceptance.
+
+## Internal details
+
+The following changes would occur:
+
+- The following files would be moved to the specification repo:
+  - `text/` directory -> `oteps/text/`
```

Comment:
People that are not part of otel community sometimes assume that OTEPs have power similar to the spec. If OTEPs are moved to the spec repo, it'd be even more subtle. 

I'd prefer if we put them in the `proposals/`, `designs/`, or other directory that makes the status obvious (and does not rely on the reader knowledge of what the `OTEP` is)

As an alternative or an addition, we could add a short warning/explanation at the beginning of each OTEP that it represents the original design decision and the spec is the actual source of truth.

Reply:
I would prefer to keep the directory name as oteps, as that is our official name for the RFC process. The README in the directory can emphasize the point that these are proposals, not the official spec. 

+1 to have the same disclaimer repeated in the OTEPs themselves, as part of the template.

Reply:
Great points both.  I think adding disclaimers to the readme and oteps themselves is suficient, I'll add that to the text here.

Reply:
I like `oteps/` with a solid README. Other names like `designs/` or `proposals/` make it sounds like after-the-fact explainer docs or things that are not really agreed upon yet.